app/frontend/src/submissions/components/SubmissionForm/WellType.vue

```diff
@@ -130,6 +154,38 @@ Licensed under the Apache License, Version 2.0 (the "License");
               :loaded="fieldsLoaded['well_identification_plate_attached']"
           ></form-input>
         </b-col>
+        <b-col cols="12" md="4" v-if="isStaffEdit">
+          <form-input
+              id="wellPlateAttachedBy"
+              label="Identification Plate Attached By"
+              type="text"
+              v-model="idPlateAttachedByInput"
+              :errors="errors['id_plate_attached_by']"
+              :loaded="fieldsLoaded['id_plate_attached_by']"
+          ></form-input>
+        </b-col>
+      </b-row>
+      <b-row v-if="isStaffEdit">
+        <b-col cols="12" md="4" v-if="isStaffEdit">
+          <form-input
+              id="waterSupplySystem"
+              label="Water Supply System Name"
+              type="text"
+              v-model="waterSupplySystemInput"
+              :errors="errors['water_supply_system_name']"
+              :loaded="fieldsLoaded['water_supply_system_name']"
+          ></form-input>
+        </b-col>
+        <b-col cols="12" md="4" v-if="isStaffEdit">
```

Comment:
It's already nested in <b-row v-if="isStaffEdit"> ; no need for each column to also check.